doc/source/kubecluster2.rst

```diff
@@ -0,0 +1,136 @@
+Dask Operator
+===========
+
+.. warning::
+    `KubeCluster2` is experimental for now. So any bug reports are appreciated!
+
+:doc: The Dask Operator is for creating and managing a Dask Cluster. It's installed on the Kubernetes cluster and then users can create clusters via the Kubernetes API (``kubectl``) or the Python API (``KubeCluster2``)
+
+Installing the Operator
+-----------------------
+
+.. currentmodule:: dask_kubernetes
+
+First you must install the Dask Operator and have access to a Kubernetes Cluster. To install the the operator, you'll need to deploy the custom resources
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/customresources/daskcluster.yaml
+   kubectl apply -f dask_kubernetes/operator/customresources/daskworkergroup.yaml
+
+Then, you'll need to create the operator
+
+.. code-block:: bash
+   kubectl apply -f dask_kubernetes/operator/deployment/manifest.yaml
+   
+Creating a Dask cluster via `kubectl`
+-------------------------------------
+
+To create a cluster in the default namespace, run the following
+
+Create a file called `cluster.yaml` and provide it with the following configuration
+
+.. code-block:: yaml
+
+   apiVersion: kubernetes.dask.org/v1
+   kind: DaskCluster
+   metadata:
+     name: simple-cluster
+   spec:
+     imagePullSecrets: null
+     image: "daskdev/dask:latest"
+     imagePullPolicy: "IfNotPresent"
+     protocol: "tcp"
+     scheduler:
+       resources: {}
+       env: {}
+       serviceType: "ClusterIP"
+     replicas: 3
+     resources: {}
+     env: {}
+
+Editing this file will change the default configuration of you Dask cluster. Now deploy `cluster.yaml`
```

Comment:
It would be great to really break down and go into detail about what each of these configuration items do. To avoid confusing users here we could add another section before the `API` section for `Configuration Reference` or something that really details all the config options. Then link it from here.